scout/server/blueprints/dashboard/templates/dashboard/dashboard_general.html

```diff
@@ -159,29 +159,41 @@ <h1>Pedigree info</h1>
   <script src="{{ url_for('dashboard.static', filename='charts.js') }}"></script>
   <script type="text/javascript">
 
-    document.querySelector("#download_verified").addEventListener("submit", function(e){
-      // Avoid page spinner being stuck on download verified variants
-      $(window).unbind('beforeunload');
-    });
+    if(document.getElementById('download_verified')){
+      document.querySelector("#download_verified").addEventListener("submit", function(e){
+        // Avoid page spinner being stuck on download verified variants
+        $(window).unbind('beforeunload');
+      });
+    }
 
-    var sel = document.getElementById("search_type");
-    var text = document.getElementById("search_term");
```

Comment:
Ah, that one is still used down below - like line 197 in hide_div - search and replace! 😁

Reply:
Fixed!

<img width="1420" alt="image" src="https://user-images.githubusercontent.com/28093618/157266325-6b1acfc5-3242-4f0b-85cf-608e9aa7e6a4.png">


<img width="1418" alt="image" src="https://user-images.githubusercontent.com/28093618/157266362-220b6532-be0e-4902-bb6a-47d2e8cb85c5.png">